src/nora/components/NoraSnackbar/NoraSnackbar.test.js

```diff
@@ -29,6 +29,7 @@ describe('NoraSnackbar', () => {
       ariaLabelledBy,
       isOpen,
       onDismiss: onDismissMock,
+      className: 'foobar',
```

Comment:
These were unrelated hotfixes for className for Drawer and NoraDrawer which resulted in error/warnings when you run `yarn test` -- just seemed easy enough to clean up here